qbicsoftware/sample-notificator-cli
# Send email containing failedQC Notification to the subscribers

**Description of changes**
Introduces the `EmailGenerator` class, which employs the linux native `mailutils` via a processbuilder to send the generated notifications to their associated subscribers. 

**Technical Information** 
For now the generated email is a simple textbased email, this could be enhanced via a dedicated html template in the future, which is possible in mailutils. 

**Additional Information**
For now the email is send to my university work address since we don't have sensible entries in our database for testing purposes.(This is marked with a ToDo)

**How to test**
1.) Change my email address with yours for testing. 
2.) run mvn clean package
3.) Run program as instruced in the readme providing a date for which notifications are stored in the database e.g.(2021-08-17)
4.) an Email with the Subject Project Update should be arriving in your email
